src/reducers/images.js

```diff
@@ -88,6 +92,15 @@ const images = (state = initialState, { type, payload }) => {
         gifFileName: payload.gifFileName
       };
 
+    case UPDATE_TEXT_POSITION:
+      return {
+        ...state,
+        ...{
```

Comment:
No need to spread a second object, just include the properties after the first spread.

Reply:
Can I fix the other reducer files that also spread a second object? Or should I leave it and just change this one?

Reply:
Just take care of yours, don't worry about the others for now.